src/database/schema/UserSchema.ts

```diff
@@ -135,7 +135,11 @@ class UserSchema implements BaseDbSchema {
         last_active_at = ?,
         profile_picture = ?,
         bio = ?,
-        is_banned = ?
+        is_banned = ?,
+        anon_user_info_emoji_name = ?,
+        anon_user_info_emoji_code = ?,
+        anon_user_info_color_name = ?,
+        anon_user_info_color_code = ?
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding new fields related to anonymous user information in the SQL query is fine as long as these fields exist in the database and are properly handled elsewhere in the code.
```diff
         bio = ?,
-        is_banned = ?
+        is_banned = ?,
+        anon_user_info_emoji_name = ?,
+        anon_user_info_emoji_code = ?,
+        anon_user_info_color_name = ?,
+        anon_user_info_color_code = ?
       WHERE
         channel_id = ? AND user_id = ?`;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->